dashboard/test/lib/safe_browsing_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class SafeBrowsingTest < ActiveSupport::TestCase
+  def test_determine_safe_website
+    assert SafeBrowsing.determine_safe_to_open("https://code.org/")
+  end
+
+  def test_determine_unsafe_website
+    refute SafeBrowsing.determine_safe_to_open("http://testsafebrowsing.appspot.com/s/phishing.html")
```

Comment:
It'd be best if this test didn't have to actually hit Google's API every time we ran it.  Can we set up VCR here?